raiden/log_config.py

```diff
@@ -44,7 +61,16 @@ def configure_logging(
     formatter = 'colorized' if colorize and not log_file else 'plain'
     if log_json:
         formatter = 'json'
-    log_handler = _get_log_handler(formatter, log_file)
+    log_handler = _get_log_handler(
+        formatter,
+        log_file,
+        logger_level_config.get('', DEFAULT_LOG_LEVEL),
+    )
+    debug_log_file_handler = _get_log_file_handler()
+
+    combined_log_handlers = {}
+    combined_log_handlers.update(log_handler)
+    combined_log_handlers.update(debug_log_file_handler)
```

Comment:
Totally unrelated to the issue but you can use the nice new 3.5+ syntax here:

`combined_log_handers = {**log_handler, **debug_log_file_handler}`